src/omt/bitvector_optimizer.cpp

```diff
@@ -0,0 +1,242 @@
+/*********************                                                        */
+/*! \file bitvector_optimizer.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Yancheng Ou
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ ** in the top-level source directory and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Optimizer for BitVector type
+ **/
+
+#include "bitvector_optimizer.h"
+
+#include "options/smt_options.h"
+#include "smt/smt_engine.h"
+#include "theory/quantifiers/quantifiers_attributes.h"
+#include "theory/smt_engine_subsolver.h"
+
+using namespace CVC4::theory;
+
+namespace CVC4::smt {
```

Comment:
ok I could change this...

Reply:
Err... sorry I would not change this... because the files are heavily referencing stuffs in CVC4::smt, changing this will introduce a lot of CVC4::smt::... in the header files. In other words, it's not completely independent from CVC4::smt. 
And introducing a new namespace like this doesn't bring us any real benefit...

Reply:
OK, it seems that only stuffs like OptResult and ObjectiveType from CVC4::smt are referred... I could change this...